docs/source/devguides/walkthroughs/VOLTTRON-Central-Demo.rst

```diff
@@ -150,7 +150,7 @@ details on how to configure the agent for your specific use case.
 Log In
 ------
 
-To log in to VOLTTRON Central, navigate in a browser to localhost:8080,
+To log in to VOLTTRON Central, navigate in a browser to localhost:8080/vc,
```

Comment:
should be localhost:8080/vc/index.html

redirection doesn't work now :(

Reply:
Done